domain/src/main/java/camp/nextstep/edu/github/domain/RepositoryVO.kt

```diff
@@ -0,0 +1,6 @@
+package camp.nextstep.edu.github.domain
+
+data class RepositoryVO(
+    var fullName: String? = null,
+    var desc: String? = null
```

Comment:
```suggestion
    val fullName: String? = null,
    val desc: String? = null
```
var을 사용하게 되면, 외부에서 값을 변경할 수 있어서 캡슐화가 깨지기 쉬워요.
val로 변경해보면 어떨까요?